lib/playback.test.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const { withPlayback } = require('../index.js');
+
+const options = {
+    dev: true,
+    isServer: true,
+};
+
+const mockFunction = jest.fn();
+
+let config;
+
+beforeEach(() => {
+    config = {
+        module: {
+            rules: [],
+        },
+    };
+
+    jest.clearAllMocks();
+});
+
+it('should match snapshot', () => {
+    const arrayOfPlaybackRules = [];
+
+    arrayOfPlaybackRules.push(withPlayback()().webpack(config, options).module.rules);
+
+    expect(arrayOfPlaybackRules).toMatchSnapshot();
+});
+
+it('should return a single rule', () => {
```

Comment:
This test seems unnecessary since the test above already tests the whole rules array shape.